fleet_vehicle_inspection/__manifest__.py

```diff
@@ -6,7 +6,7 @@
     'summary': """
         This module extends the Fleet module allowing the registration
         of vehicle entry and exit inspections.""",
-    'version': '12.0.1.0.0',
+    'version': '12.0.1.1.0',
```

Comment:
In the future, there is no need to bump the version number, as that's already managed by @OCA-git-bot when merging.
The only case where you want to bump it is when you include migration scripts and you need to make sure they have the same version number

Anyway, not an issue :)

Reply:
Thanks